### PR TITLE
Changed emacs-mac-port link

### PR DESCRIPTION
**Pull request by CarlQLange**
Changed it from the now-un-updated mirror to the correct homebrew package.

### CODE REVIEW

**Comment by stormpat** [comment]
This fixes #857

File: README.md [hunk 1]
```diff
@@ -500,7 +500,7 @@ details.
 [evil-nerd-commenter]: https://github.com/redguardtoo/evil-nerd-commenter
 [Gitter Chat]: https://gitter.im/syl20bnr/spacemacs
 [MacType]: https://code.google.com/p/mactype/
-[emacs-mac-port]: https://github.com/railwaycat/emacs-mac-port
+[emacs-mac-port]: https://github.com/railwaycat/homebrew-emacsmacport
 [emacs-mac-port-server]: https://github.com/railwaycat/emacs-mac-port/blob/master/README-mac#L210-L213
```
Comment (ccmtaylor) [inline README.md]: should this also point somewhere else?

Comment (CarlQLange) [inline README.md]: Nope, the readme is still accurate and is not on the other repo.

Comment (ccmtaylor) [inline README.md]: :+1:

**Comment by syl20bnr** [comment]
Oops I merged this PR but did not update the thread.

@CarlQLange Thank you !
Cherry-picked in `develop` branch.

**Pull request closed without merging**